opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -55,26 +52,88 @@
 
 _EXPORTER_OTLP = "otlp"
 _EXPORTER_OTLP_PROTO_GRPC = "otlp_proto_grpc"
+_EXPORTER_OTLP_PROTO_HTTP = "otlp_proto_http"
 
 _RANDOM_ID_GENERATOR = "random"
 _DEFAULT_ID_GENERATOR = _RANDOM_ID_GENERATOR
 
+_logger = logging.getLogger(__name__)
+
 
 def _get_id_generator() -> str:
     return environ.get(OTEL_PYTHON_ID_GENERATOR, _DEFAULT_ID_GENERATOR)
 
 
-def _get_exporter_names(names: str) -> Sequence[str]:
-    exporters = set()
+def _get_otlp_exporter_entry_point(
+    otlp_protocol: Optional[str],
+) -> Literal[_EXPORTER_OTLP_PROTO_GRPC, _EXPORTER_OTLP_PROTO_HTTP]:
+    if otlp_protocol == "grpc":
+        return _EXPORTER_OTLP_PROTO_GRPC
+    if otlp_protocol == "http/protobuf":
```

Comment:
I might be missing something, but shouldn't a case for "http/json" be included as well?

Reply:
Only HTTP over protobuf is currently available in Python
(https://github.com/open-telemetry/opentelemetry-python/blob/main/exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/trace_exporter/__init__.py)